service/routes.py

```diff
@@ -80,6 +80,19 @@ def get_recommendations(product_id1, product_id2):
         raise NotFound("Recommendation for product id {} and {} was not found.".format(product_id1, product_id2))
     return make_response(jsonify(recommendation.serialize()), status.HTTP_200_OK)
 
+@app.route("/recommendations/<int:product_id1>/related-products/<int:product_id2>", methods=["DELETE", "GET"])
```

Comment:
Actually I figured out. You might be using the path for GET to test, which is  
 /recommendations/**products**/<int:product_id1>/related-products/<int:product_id2>. 
While the uri you listed here is missing the "products" in the path. 
I think you could add the /products in the path and then you should be able to remove "GET" in this route.

Reply:
Yes it is. That's the problem. Thanks a lot I get it fix and I'm ready to merge now.